src/gui/touchscreengui.cpp

```diff
@@ -466,6 +466,7 @@ void TouchScreenGUI::init(ISimpleTextureSource *tsrc)
 {
 	assert(tsrc);
 
+	m_initialized   = true;
```

Comment:
Maybe move this to the end of the function in case of an exception during call of this function?

Reply:
Sure, why not.